gnomad/sample_qc/ancestry.py

```diff
@@ -129,6 +219,10 @@ def assign_population_pcs(
     output_col: str = "pop",
     missing_label: str = "oth",
     pc_expr: Union[hl.expr.ArrayExpression, str] = "scores",
+    convert_model_func: Optional[Callable[[Any], Any]] = None,
+    apply_model_func: Callable[
+        [pd.DataFrame, Any], Any
+    ] = apply_sklearn_classification_model,
```

Comment:
No, it's because the model is too old, so you need to use old versions of sklearn and other packages in order to loaded it. That is why we are updating to ONNX and why I didn't test the v2.1 RF model. Even though the v3.1 RF model loads, it still loads with 

```UserWarning: Trying to unpickle estimator RandomForestClassifier from version 0.23.2 when using version 0.24.2. This might lead to breaking code or invalid results. Use at your own risk.
```
Like mentioned in the users reported issue in [#533](https://github.com/broadinstitute/gnomad_methods/issues/533)

Reply:
Okay, so you converted  them to ONNX version separately with older sklearn? Are you going to share the two *.onnx in the public bucket? The test in your notebook shows your functions are working to me.

Reply:
Could you also add a note about the #533 issue? So when someone tries to load the sklearn RF model, they are aware of this issue, they may use the *.onnx model directly if their python is more recent.

Reply:
> Okay, so you converted them to ONNX version separately with older sklearn? 

Yeah, I finally got the configurations needed for the v2.1 and v3.1 sklearn models to each load and converted them to ONNX models.

> Are you going to share the two *.onnx in the public bucket? 

That's the plan, but the first step was to make these functions and get them merged. Then I also have tickets to:

[Add an example of gnomAD ancestry RF model use to `gnomad_qc`](https://app.zenhub.com/workspaces/gnomad-5f4d127ea61afc001d6be50b/issues/gh/broadinstitute/gnomad_production/1038)

[Modify blog post on use of ancestry RF model to link to `gnomad_qc` example](https://app.zenhub.com/workspaces/gnomad-5f4d127ea61afc001d6be50b/issues/gh/broadinstitute/gnomad_production/1039)

The idea is that the ONNX models will replace the sklearn models, and the blog post will be updated with no code, but instead a link to an example in `gnomad_qc` so if we need to make changes to it, we can do that in `gnomad_qc` and not need to modify the blog post again.

> Could you also add a note about the https://github.com/broadinstitute/gnomad_methods/issues/533 issue? So when someone tries to load the sklearn RF model, they are aware of this issue, they may use the *.onnx model directly if their python is more recent.

I'm not sure what you mean? I can add this note to the `gnomad_qc` example (when I have made it), and mention it in the change to the blog post, but I don't think there is a good place for this note in the `gnomad_methods` code

Reply:
You answered my last question in your plan. I understand gnomad_methods is more general. 
I think it is good to go!